mailgun/templates/mailgun/email/bounce_back_not_subscribed.html

```diff
@@ -0,0 +1,20 @@
+<html>
+    <head>
+        <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+        <meta name="viewport" content="width=device-width">
+    </head>
+    <body>
+        <h4>***Your message could not be delivered to mailing list {{ recipient }}.***</h4>
+        <p>
+            This mailing list does not recognize the email address you are using. This could be because 1) you are not
```

Comment:
I know this matches the text in the google doc, but it seems like it should be `because 1)..., or 2) ...`, or `1) because ..., or 2) because ...`.  The way it is now (`because 1)..., or 2) because ...`) is just awkward.